src/gui/screen_version_info.cpp

```diff
@@ -117,4 +118,6 @@ screen_t screen_version_info = {
     0,                          //pdata
 };
 
-const screen_t *pscreen_version_info = &screen_version_info;
+extern "C" {
+screen_t *pscreen_version_info = &screen_version_info;
+}
```

Comment:
You can write `extern "C" screen_t ...`, no need to have it in a block.

Reply:
(this applies to more files in the PR)

Reply:
fixed